dts/x86/intel/alder_lake.dtsi

```diff
@@ -29,6 +29,8 @@
 	};
 
 	intc: ioapic@fec00000  {
+		#address-cells = <1>;
+		#size-cells = <1>;
```

Comment:
Do you really need `#size-cells` too? I found one quite old PR which fixed the same thing for another interrupt controller: #36499

Reply:
And here's a more recent commit for riscv: https://github.com/zephyrproject-rtos/zephyr/commit/07ac630281c353c03b2b147f69dc8ff30a0fc357

Reply:
And another even more recent PR: #55803